src/model/model.ts

```diff
@@ -3544,6 +3563,68 @@ class Model {
         }
         return ProdServerEndpoint;
     }
+    
+    parseKeyDescription(description: string): Error | KeyPressType {
```

Comment:
unfortunately need to do this differently to support multiple mod keys, like ctrl + shift + c

I think I will use | for or, so ctrl + shift + c = "ctrl:shift:c"
but if we want either ctrl or shift and c, it will be ctrl|shift:c, and you could even do ctrl|opt|cmd:shift:c, etc

Then I think I will parse to an array, so ctrl:shift:c = ["ctrl", "shift", "c"], and ctrl|shift|cmd:c = [[ctrl, shift, cmd], "c"]

I think this makes sense, it's def a little complex tho

Reply:
This is now implemented this way

Reply:
this has changed